Script/AtomicEditor/editor/Editor.ts

```diff
@@ -77,7 +77,7 @@ class Editor extends Atomic.ScriptObject {
                 playerWindow.monitor = data.monitor;
                 playerWindow.maximized = true;
             } else {
-                playerWindow = {x: data.posX, y: data.posY, width: data.width, height: data.height, monitor: data.monitor, maximized: data.maximized};
+                playerWindow = {x: data.posX, y: data.posY, width: data.width, height: data.height, monitor: data.monitor, maximized: data.maximized, windowed: data.windowed};
```

Comment:
Does a "windowed" event parameter need to be set here? https://github.com/AtomicGameEngine/AtomicGameEngine/blob/master/Source/AtomicApp/Player/IPCPlayerApp.cpp#L270

(BTW, I think "fullscreen" may be a little less confusing and is consistent with the term used in the Graphics API)